### PR TITLE
Sticky nav

### PR DESCRIPTION
**Pull request by hippogriffic**
image

needs some work to make properly sticky, but I'd let to get this merged for now as it fixes the whole repo (which I apparently broke)

@incuna/frontend pls

### CODE REVIEW

File: sass/components/_section-nav.sass [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    padding: 14px
+    border:
+        bottom: 1px solid $grey-4
+    cursor: pointer
+    position: relative
+
+    &::after
+        @include transition(all 0.2s ease-out)
+        @include transform(rotate(0))
+        content: '∇'
```
Comment (pandalion) [inline sass/components/_section-nav.sass:30]: beak

File: sass/components/_section-nav.sass [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        line-height: 1
+        color: $grey-5
+        position: absolute
+        right: 14px
+        top: 15px
+
+    #{$hover-states},
+    &.open
+        background: $grey-2
+
```
Comment (pandalion) [inline sass/components/_section-nav.sass:43]: why this blank line looks a bit weird

File: scripts/style-guide.js [hunk 3]
```diff
@@ -3,9 +3,15 @@
...
     $(document).ready(function () {
         // Add a class 'toggle-item' to elements to have them toggle a class of 'open' when clicked (to demonstrate show/hide effects etc)
-        $(document).on('click', '.toggle-item', function (e) {
+        $(document).on('click', '.js-toggle-item', function (e) {
             e.preventDefault();
             $(this).toggleClass('open');
         });
+
+        $('.js-toggle-header').click(function () {
+            $(this).toggleClass('open');
```
Comment (pandalion) [inline scripts/style-guide.js]: does it work to do
` $(this).toggleClass('open').next('.js-toggle-body').toggleClass('open');` ?
I don't remember

File: sass/components/_section-nav.sass (hunk 2 above)
Comment (hippogriffic) [reply sass/components/_section-nav.sass:43]: why?

Comment (pandalion) [reply sass/components/_section-nav.sass:43]: why not

**Comment by pandalion** [comment]
Looks good! 👍

File: scripts/style-guide.js (hunk 3 above)
Comment (hippogriffic) [reply scripts/style-guide.js]: no idea, I just grabbed this from somewhere else 😁

File: sass/components/_section-nav.sass (hunk 2 above)
Comment (hippogriffic) [reply sass/components/_section-nav.sass:43]: I like it :P

Comment (pandalion) [reply sass/components/_section-nav.sass:43]: I like your face

File: scripts/style-guide.js (hunk 3 above)
Comment (hippogriffic) [reply scripts/style-guide.js]: you are correct, it does work

**Comment by pandalion** [comment]
yay

**Comment by hippogriffic** [comment]
thaaaanks

**Pull request merged**